examples/typescript/src/App.tsx

```diff
@@ -9,10 +9,21 @@ import {
   MessageInput,
   Thread,
   Window,
+  useCreateChatClient,
 } from 'stream-chat-react';
 
 import './App.css';
 
+declare module 'stream-chat' {
+  interface UserEx {
+    customUserField: string;
+  }
+
+  interface ReactionEx {
+    customReactionField: string;
+  }
+}
```

Comment:
This is what integrators will have to do, instead of using SCG.

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWQzd3NpbWZvNmNxMGxkc3R2N3J6Njl1dGU0ZGZpNmgwYmMyZ3hqYSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3o8doT9BL7dgtolp7O/giphy.gif)

Reply:
This looks very nice! 🚀  Let's plan to fix this for every JS SDK